Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM node:12.19.0-alpine3.12 as builder
 
 ENV NODE_ENV development
-ENV NODE_PATH=src/
```

Comment:
This shouldn't be part of this PR.

Reply:
I'm not clear on the purpose of setting this ENV_VAR. It is probably worth while checking in with devops to see why it was set and what it is used for